content/en/user-guide/aws/apigateway/index.md

```diff
@@ -206,6 +206,42 @@ $ curl -X GET http://localhost:4566/restapis/<REST_API_ID>/test/_user_request_/t
 {"message":"Hello World"}
 {{< /command >}}
 
+## New API Gateway implementation
+
+{{< callout >}}
+
+Since [3.8.0](link here), LocalStack supports a new API Gateway implementation for both API Gateway v1 (REST API) and v2 (HTTP API).
```

Comment:
Just a reminder to not forget to update this link before merging. I also added a TODO on the PR comment